data/modules/Combat/Combat.lua

```diff
@@ -216,7 +217,17 @@ local makeAdvert = function (station)
 
 	location = flavour.planets[Engine.rand:Integer(1, #flavour.planets)]
 	dist = location:DistanceTo(Game.system)
+
+	local tradeCargo = Game.player.totalCargo*0.81 --the rest 0.19 is used for fuel.
+	local rewardRatio = 3/4-- For getting that ratio of average effective trading profit
+
+	--- Fighters should have increased reward. For that we increase minimum reward depending on number of missiles that player can acquire.
+	local missiles = Game.player:GetEquipSlotCapacity("missile")
+	local minimumRewardRatio = math.max(rewardRatio, Calibration.Fighting(missiles))
+
+	local profit = math.max(Calibration.profit(tradeCargo) * rewardRatio, Calibration.minProfit * minimumRewardRatio)
```

Comment:
Same remark, cargo space has nothing to do with combat!

Reply:
Same reply :)